season-palette.lua

```diff
@@ -0,0 +1,221 @@
+-- Swap color palettes with the changes of the seasons.
+-- 
+-- Based on my script for the Rubble addon "Libs/Colors/Swap Palette", modified at
+-- Meph's request to support automatic seasonal switching.
+
+--[[
+Copyright 2016 Milo Christiansen
+
+This software is provided 'as-is', without any express or implied warranty. In
+no event will the authors be held liable for any damages arising from the use of
+this software.
+
+Permission is granted to anyone to use this software for any purpose, including
+commercial applications, and to alter it and redistribute it freely, subject to
+the following restrictions:
+
+1. The origin of this software must not be misrepresented; you must not claim
+that you wrote the original software. If you use this software in a product, an
+acknowledgment in the product documentation would be appreciated but is not
+required.
+
+2. Altered source versions must be plainly marked as such, and must not be
+misrepresented as being the original software.
+
+3. This notice may not be removed or altered from any source distribution.
+]]
+
+--@ module = true
+--@ enable = true
+
+local mapingsA = {
+	["BLACK"] = 0,
```

Comment:
Omitting the brackets results in the keys being strings (except you're essentially restricted to valid identifiers as keys, which isn't an issue here). I suppose it doesn't really matter, though.

This is fine in the main folder because it can be used directly by end-users. `modtools` is only for mod-oriented utilities that users shouldn't be running directly (well, except for testing purposes).